crocket/src/main/java/com/crocket/Ball.java

```diff
@@ -0,0 +1,30 @@
+package com.crocket;
+public class Ball extends Entity implements IMovable{
+    private double mass;
+    private String powerUp;
+    private double xVelocity;
+    private double yVelocity;
+
+    public Ball(int width, int height, double xPosition, double yPosition, double mass) {
+        super(width, height, xPosition, yPosition);
+        //TODO Auto-generated constructor stub
+    }
+
+    public double getxVelocity(){
+        return xVelocity;
+    }
+
+    public double getyVelocity(){
+        return yVelocity;
+    }
+
+    public void startBall(double cosinus, double sinus, double power){
```

Comment:
I do not know but we might want to check if mass is 0.

Reply:
Mass should never be 0 unless we give the ball a mass of 0 when we make an new instance of a ball object.

Reply:
Added a check in the constructor